feincms_extensions/tests/content.py

```diff
@@ -8,3 +8,6 @@ class Meta:
 
     def render(self, **kwargs):
         return ''
+
+    def json(self, **kwargs):
+        return ''
```

Comment:
Looks good. I was thinking we could use something similar to the `rest-framework` and use a serialiser here.